### ISSUE TITLE
[Bug] [Android] [a11y] Semantic Headings are not accessible or functional

### ISSUE THREAD

**Issue by rachelkang** [dotnet/maui#1902]
### Description

Semantic headings are not accessible (appear not to be in the accessibility tree) or functional (not accessible via Headings mode either).

### Steps to Reproduce

1. Launch .NET MAUI Android sample on an Android device
2. Navigate to SemanticsPage
3. Turn on TalkBack
4. Swipe to labels w/ SemanticHeading towards the bottom
5. Observe buggy behavior - semantic focus does not go to headings (to test heading, three finger swipe rotor to switch to headings mode)

### Expected Behavior
Labels with SemanticProperties.HeadingLevel should receive semantic focus / be in the accessibility tree unless otherwise specified
Labels with SemanticProperties.HeadingLevel should be reached in heading mode

### Actual Behavior
Labels with SemanticProperties.HeadingLevel are not receiving semantic focus / don't appear to be in the accessibility tree
Labels with SemanticProperties.HeadingLevel are not reached in heading mode

### Screenshots

https://user-images.githubusercontent.com/21988533/127912392-52748094-2573-45c9-a2a8-1a4b110bb0fa.mp4

**Comment by rachelkang** [comment]
related: https://github.com/xamarin/XamarinCommunityToolkit/issues/1543

**Comment by rachelkang** [comment]
Closing as this issue was fixed and is no longer an issue.
